contents/security/crossSiteRequestForgery/crossSiteRequestForgery.md

```diff
@@ -0,0 +1,96 @@
+# Cross Site Request Forgery (CSRF)
+
+Author: [Tran Tien Dat](https://github.com/tran-tien-dat)
+
+CSRF is a dangerous type of attack that has affected major sites like [Gmail](http://archive.oreilly.com/pub/post/gmail_exploit_contact_list_hij.html) or [Netflix](http://blog.jeremiahgrossman.com/2006/10/more-on-netflixs-csrf-advisory.html) in the past. This article attempts to give an easy-to-digest introduction to the attack and how to protect your website from it.
+
+## Example
+
+Cross-Site Request Forgery (CSRF) is an attack that tricks the victim to perform unwanted actions on a web application in which they're currently authenticated. 
```

Comment:
I think "unwanted actions" can be better phrased. As a first-time reader, I would be wondering what you mean by unwanted actions. Unwanted by who? Why are such actions unwanted?